src/S3/S3Client.php

```diff
@@ -237,46 +237,6 @@ public function copyObject($input = []): CopyObjectOutput
         return new CopyObjectOutput($response);
     }
 
-    /**
-     * @see http://docs.amazonwebservices.com/AmazonS3/latest/API/RESTObjectGET.html
-     *
-     * @param array{
-     *   Bucket: string,
-     *   IfMatch?: string,
-     *   IfModifiedSince?: \DateTimeImmutable|string,
-     *   IfNoneMatch?: string,
-     *   IfUnmodifiedSince?: \DateTimeImmutable|string,
-     *   Key: string,
-     *   Range?: string,
-     *   ResponseCacheControl?: string,
-     *   ResponseContentDisposition?: string,
-     *   ResponseContentEncoding?: string,
-     *   ResponseContentLanguage?: string,
-     *   ResponseContentType?: string,
-     *   ResponseExpires?: \DateTimeImmutable|string,
-     *   VersionId?: string,
-     *   SSECustomerAlgorithm?: string,
-     *   SSECustomerKey?: string,
-     *   SSECustomerKeyMD5?: string,
-     *   RequestPayer?: string,
-     *   PartNumber?: int,
-     * }|GetObjectRequest $input
-     */
-    public function getObject($input = []): GetObjectOutput
```

Comment:
Is their a way to tell avoid such diff?
- tell to `nette/php-generator` to sort methods?
- add a php-cs-fixer on `/*/*Client` to sort methods?

Reply:
> tell to nette/php-generator to sort methods?

No, and there is a massive "copy code" to achieve that. 

> add a php-cs-fixer on /*/*Client to sort methods?

I dont know. Can php-cs-fixer sort? Why not sort everything then?